test/Codenizer.HttpClient.Testable.Tests.Unit/WhenMatchingRoutes.cs

```diff
@@ -64,6 +72,53 @@ public void GivenNonConfigured_NoResultIsReturned()
                 .Should()
                 .BeNull();
         }
+        
+        
+        [Fact]
+        public void GivenRouteWithHeaders_RequestBuilderIsReturned()
+        {
+            var requestBuilder = new RequestBuilder(HttpMethod.Get, "/api/foos/{id}", null);
+            requestBuilder.Authorization(new AuthenticationHeaderValue("Bearer"));
+
+            var routes = new List<RequestBuilder>
+            {
+                requestBuilder
+            };
+
+            var dictionary = ConfiguredRequests.FromRequestBuilders(routes);
+            
+            dictionary
+                .Match(
+                    HttpMethod.Get,
+                    "/api/foos/1234", 
+                    null,
+                    new AuthenticationHeaderValue("Bearer"))
+                .Should()
+                .Be(requestBuilder);
+        }
+        
+        
+        [Fact]
+        public void GivenRouteWithHeadersAndEmptyRequestHeaders_RequestBuilderIsNotReturned()
```

Comment:
Could you change the name of this test method too? Same as with the previous one.